CHANGELOG-1.6.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+
+Changelog for K8ssandra, new PRs should update the `unreleased` section below with entries sorted by type, in the 
+following order:
+
+```markdown
+* [CHANGE]
+* [FEATURE]
+* [ENHANCEMENT]
+* [BUGFIX]
+```
+
+If two entries have the same type, they should be sorted from newest to oldest (the newest comes first, the oldest comes 
+last).
+
+When cutting a new release of the parent `k8ssandra` chart update the `unreleased` heading to the tag being generated 
+and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unreleased` entries.
+
+## unreleased
+
+* [CHANGE] Upgrade Stargate to v1.0.63
+* [CHANGE] Upgrade cass-operator to v1.10.4
+* [CHANGE] Deprecate all K8ssandra v1.x charts, refer users to k8ssandra-operator.
+* [CHANGE] Upgrade Reaper to v3.2.0
+* [ENHANCEMENT] Added s3_rgw support
+* [CHANGE] cass-operator HELM-Chart: Upgrade requires manual action if registryOverride was used before. registryOverride is now repositoryOverride.
+* [ENHANCEMENT] Enable webhook functionality in cass-operator if cert-manager is installed.
+* [CHANGE] [#1409](https://github.com/k8ssandra/k8ssandra/pull/1409) Upgrade cass-operator to v1.11.0
```

Comment:
This should be removed since cass-operator wasn't upgraded. Looks like there are some other errors as well. The changelog should only include the changes on the 1.5 branch since 1.5.1.

Reply:
Ah gosh. I don't understand where these changes come from. I literally only changed one line in my commit, but when I cherry pick that commit over onto this branch, all of this additional stuff comes in.

Reply:
Ah, I think it is maybe because there was no 1.6 changelog in the original branch... Maybe I'll delete the file from this branch and add the deprecation changes to the 1.5 changelog.